core/iwasm/include/wasm_export.h

```diff
@@ -222,6 +222,13 @@ typedef struct RuntimeInitArgs {
 #define LOAD_ARGS_OPTION_DEFINED
 typedef struct LoadArgs {
     char *name;
+    /* This option is only used by the Wasm C API (see wasm_c_api.h) */
+    bool clone_wasm_binary;
+    /* False by default, used by AOT/wasm loader only.
```

Comment:
what's AOT/wasm loader here?
aot_loader.c, wasm_loader.c, and wasm_mini_loader.c?
do you mean all module loaders?

Reply:
Yes, all module loaders basically